packages/salesforcedx-vscode-core/src/commands/isvdebugging/bootstrapCmd.ts

```diff
@@ -436,11 +396,11 @@ export class IsvDebugBootstrapExecutor extends SfdxCommandletExecutor<{}> {
         notificationService.showErrorMessage(
           nls.localize('error_writing_installed_package_info', error.toString())
         );
-        return;
+        continue;
```

Comment:
For package name like `Salesforce.com CRM Dashboard` It retrieves package within folder name `Salesforce` which is not same as package name. Eventually errors out while adding installed-package.json saying no directory named `Salesforce.com CRM Dashboard` was found.
Added continue to surpass that error. Need to discuss how to handle this edge case.

Reply:
@peternhale when retrieving a package with `.com` in its name. 
Command run:`sfdx project:retrieve:start --package-name Salesforce.com CRM Dashboards --target-metadata-dir .sfdx/tools/installed-packages --unzip --zip-file-name Salesforce.com CRM Dashboards`

Reply:
@RitamAgrawal can you try wrapping the package name in quotes

```
sfdx project:retrieve:start --package-name "Salesforce.com CRM Dashboards" --target-metadata-dir .sfdx/tools/installed-packages --unzip --zip-file-name "Salesforce.com CRM Dashboards"
```